.gitlab-ci/packet.yml

```diff
@@ -153,6 +153,11 @@ packet_almalinux8-calico:
   extends: .packet_pr
   when: on_success
 
+packet_rockylinux8-calico:
+  stage: deploy-part2
+  extends: .packet_pr
+  when: on_success
+
```

Comment:
Failed
```
TASK [bootstrap-os : Fetch /etc/os-release] ************************************
task path: /builds/kargo-ci/kubernetes-sigs-kubespray/roles/bootstrap-os/tasks/main.yml:2
Wednesday 01 June 2022  20:45:50 +0000 (0:00:00.145)       0:00:11.383 ******** 
[WARNING]: raw module does not support the environment keyword
[WARNING]: raw module does not support the environment keyword
fatal: [instance-1]: FAILED! => {"changed": false, "msg": "non-zero return code", "rc": 127, "stderr": "Warning: Permanently added '172.30.114.189' (ECDSA) to the list of known hosts.\r\nShared connection to 172.30.114.189 closed.\r\n", "stderr_lines": ["Warning: Permanently added '172.30.114.189' (ECDSA) to the list of known hosts.", "Shared connection to 172.30.114.189 closed."], "stdout": "bash: sudo: command not found\r\n", "stdout_lines": ["bash: sudo: command not found"]}
fatal: [instance-2]: FAILED! => {"changed": false, "msg": "non-zero return code", "rc": 127, "stderr": "Warning: Permanently added '172.30.113.166' (ECDSA) to the list of known hosts.\r\nShared connection to 172.30.113.166 closed.\r\n", "stderr_lines": ["Warning: Permanently added '172.30.113.166' (ECDSA) to the list of known hosts.", "Shared connection to 172.30.113.166 closed."], "stdout": "bash: sudo: command not found\r\n", "stdout_lines": ["bash: sudo: command not found"]}
```
`sudo` command is not installed on the official vm image?

Reply:
The Vagrant image of rockylinux8 contains `sudo` command:
```
[vagrant@k8s-1 ~]$ cat /etc/os-release 
NAME="Rocky Linux"
VERSION="8.6 (Green Obsidian)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="8.6"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Rocky Linux 8.6 (Green Obsidian)"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:rocky:rocky:8:GA"
HOME_URL="https://rockylinux.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
ROCKY_SUPPORT_PRODUCT="Rocky Linux"
ROCKY_SUPPORT_PRODUCT_VERSION="8"
REDHAT_SUPPORT_PRODUCT="Rocky Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8"
[vagrant@k8s-1 ~]$ 
[vagrant@k8s-1 ~]$ sudo
usage: sudo -h | -K | -k | -V
usage: sudo -v [-AknS] [-g group] [-h host] [-p prompt] [-u user]
usage: sudo -l [-AknS] [-g group] [-h host] [-p prompt] [-U user] [-u user] [command]
usage: sudo [-AbEHknPS] [-r role] [-t type] [-C num] [-g group] [-h host] [-p prompt] [-T timeout] [-u user] [VAR=value] [-i|-s] [<command>]
usage: sudo -e [-AknS] [-r role] [-t type] [-C num] [-g group] [-h host] [-p prompt] [-T timeout] [-u user] file ...
```
The image of https://download.rockylinux.org/pub/rocky/8.6/images/Rocky-8-GenericCloud-8.6-20220515.x86_64.qcow2 doesn't contain it..

Reply:
Trying to add `packages: sudo` into cloud-init.